components/badges/badges.css

```diff
@@ -0,0 +1,65 @@
+.profile-badge {
+    position: absolute;
+    height: 1rem;
+    width: 1rem;
+    bottom: 0.5rem;
+    right: 0.5rem;
+    border: 2px solid var(--bg-color);
+    border-radius: 2rem;
+}
+
+.badge-container {
+    width: max-content;
+    position: relative;
+}
+
+.status-online {
+    background-color: var(--success-color)
+}
+
+.status-idle {
+    background-color: var(--warning-color);
+}
+
+.status-do-not-disturb{
+    background-color: var(--danger-color);
+}
+
+.status-offline {
+    background-color: var(--secondary-color);
+}
+
+.status-invisible {
+    background-color: var(--bg-color);
+    border: 3.5px solid var(--secondary-color);
+}
+
+.badge-status.badge-notification {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
```

Comment:
You can make a separate utility class for this named as minwii-flex-column-centered and pass these properties inside it and use it inside wherever required.

Reply:
thanks for that suggestion, will do that.